src/app/calendar-item/calendar-item.component.spec.ts

```diff
@@ -1,25 +1,86 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { Component } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
+import { HttpClientModule } from '@angular/common/http';
+import { ModalModule } from 'ngx-bootstrap/modal';
 
 import { CalendarItemComponent } from './calendar-item.component';
+import { IssueLabelComponent } from '../issue-label/issue-label.component';
+import { DailyTimeEntryComponent } from '../daily-time-entry/daily-time-entry.component';
+import { GapComponent } from '../gap/gap.component';
+import { TimeEntriesComponent } from '../time-entries/time-entries.component';
+import { dayLog2 } from '../models/time-entries.mock';
 
 describe('CalendarItemComponent', () => {
-  let component: CalendarItemComponent;
-  let fixture: ComponentFixture<CalendarItemComponent>;
+  @Component({
+    selector: `app-host-component`,
+    template: `<app-calendar-item [dayLog]="dayLog"></app-calendar-item>`
+  })
+  class TestHostComponent {
+    dayLog = dayLog2;
```

Comment:
Use clone of data. It is always safer to use clone, because if the component that is being tested changes data, it changes also values that are used in expectation statements. So, even though data has been unwillingly changed, expectation checks pass.